prymer/offtarget/bwa.py

```diff
@@ -194,29 +188,23 @@ class BwaResult:
 MAX_GAP_OPENS: int = 0
 """The default maximum number of gap opens allowed in the full query sequence"""
 
-MAX_GAP_EXTENSIONS: int = -1
+MAX_GAP_EXTENSIONS: int | None = None
 """The default maximum number of gap extensions allowed in the full query sequence"""
 
+MIN_INDEL_TO_END_DISTANCE: int = 3
+"""Do not place an indel within this many bp of the ends of the query sequence"""
+
 SEED_LENGTH: int = 20
 """The default length of the seed region"""
 
 BWA_AUX_EXTENSIONS: list[str] = [".amb", ".ann", ".bwt", ".pac", ".sa"]
 """The file extensions for BWA index files"""
 
 
-class BwaAlnInteractive(ExecutableRunner):
-    """Wrapper around a novel mode of 'bwa aln' that allows for "interactive" use of bwa to keep
-    the process running and be able to send it chunks of reads periodically and get alignments
-    back without waiting for a full batch of reads to be sent.
-
-    See:
-
-    - [https://github.com/fulcrumgenomics/bwa-aln-interactive](https://github.com/fulcrumgenomics/bwa-aln-interactive)
-    - [https://bioconda.github.io/recipes/bwa-aln-interactive/README.html](https://bioconda.github.io/recipes/bwa-aln-interactive/README.html)
+class Bwa:
```

Comment:
You can delete the entirety of `ExecutableRunner` and associated tests now too :)